aspnetcore/blazor/host-and-deploy/server.md

```diff
@@ -96,55 +96,94 @@ Consider using the [Azure SignalR Service](xref:signalr/scale#azure-signalr-serv
+
+   * Set the option in `Program.cs`:
+
+     ```csharp
+     builder.Services.AddSignalR().AddAzureSignalR(options =>
+     {
+         options.ServerStickyMode = 
+             Microsoft.Azure.SignalR.ServerStickyMode.Required;
+     });
+     ```
+
+   * Configure the option in `appsettings.json`:
+
+       ```json
+       "Azure:SignalR:ServerStickyMode": "Required"
+       ```
+
+   * Configure the option with an application setting in the Azure portal (**Configuration** > **Application settings**: **Name**: `Azure__SignalR__ServerStickyMode`, **Value**: `Required`).
+  
+  > [!NOTE]
+  > The following error is thrown by an app that hasn't enabled session affinity:
+  >
+  > > :::no-loc text="blazor.server.js:1 Uncaught (in promise) Error: Invocation canceled due to the underlying connection being closed.":::
 
-Provisioning the Azure SignalR Service in Visual Studio automatically [enables *sticky sessions*](#configuration) and adds the SignalR connection string to the app service's configuration.
+1. Use the following guidance to configure the service:
+
+  * [Configure the app in Azure App Service](xref:signalr/publish-to-azure-web-app#configure-the-app-in-azure-app-service).
+  * [App Service Plan Limits](xref:signalr/publish-to-azure-web-app#app-service-plan-limits).
 
 :::moniker range=">= aspnetcore-6.0"
 
-### Scalability on Azure Container Apps
+## Azure Container Apps
 
-Scaling server-side Blazor apps on Azure Container Apps requires specific considerations in addition to using the [Azure SignalR Service](#azure-signalr-service). Due to the way request routing is handled, the ASP.NET Core data protection service must be configured to persist keys in a centralized location that all container instances can access. The keys can be stored in Azure Blob Storage and protected with Azure Key Vault. The data protection service uses the keys to deserialize Razor components.
+For a deeper exploration of scaling server-side Blazor apps on the Azure Container Apps service, see <xref:host-and-deploy/scaling-aspnet-apps/scaling-aspnet-apps>. The tutorial explains how to create and integrate the services required to host apps on Azure Container Apps. Basic steps are also provided in this section.
 
-> [!NOTE]
-> For a deeper exploration of this scenario and scaling container apps, see <xref:host-and-deploy/scaling-aspnet-apps/scaling-aspnet-apps>. The tutorial explains how to create and integrate the services required to host apps on Azure Container Apps. Basic steps are also provided in this section.
+1. Configure *session affinity*, also known as *sticky sessions*, where clients are redirected back to the same server. Session affinity is configured by setting the Azure SignalR `ServerStickyMode` option to `Required`. Typically, an app creates the configuration using **one** of the following approaches:
```

Comment:
Again, this bit looks like it's for configuring the Azure SignalR service, which isn't needed or recommended when hosting on Azure Container Apps. There shouldn't be an Azure SignalR service content in either the Azure App Service or Azure Container App sections.